javascript-and-json/main.js

```diff
@@ -0,0 +1,34 @@
+var arr = [
+  {
+    isbn: 111111,
+    title: 'Harry Potter',
+    author: 'J.K. Rowlings'
+  },
+  {
+    isbn: 222222,
+    title: 'Lord of the Rings',
+    author: 'J.R.R. Tolkien'
+  },
+  {
+    isbn: 333333,
+    title: '1984',
+    author: 'George Orwell'
+  }
+];
+
+for (let i = 0; i < arr.length; i++) {
+  var key;
+  for (key in arr[i]) {
+    console.log(key + ': ', arr[i][key]);
+    console.log('typeof: ', typeof arr[i][key]);
+  }
+}
+
+var books = JSON.stringify(arr);
+console.log('typeof ' + books + ': ', typeof books);
+
+var a = JSON.stringify({ id: 1234, name: 'Chris Ahn' });
```

Comment:
here you don't want to use `stringify` you want to manually create a JSON string

Reply:
The instruction said to use stringify so do I make my array objects into strings manually?